src/html-conventions-lib/data-type-html-conventions.xsl

```diff
@@ -90,15 +90,20 @@
             <xsl:call-template name="nonPublicElement">
                 <xsl:with-param name="element" select="."/>
             </xsl:call-template>
+            
+            <xsl:call-template name="elementUniqueName">
+                <xsl:with-param name="element" select="."/>
+                <xsl:with-param name="isAttribute" select="fn:false()"/>
+            </xsl:call-template>
             <!--    End of common checkers rules     -->   
             <!--    Start of specific checker rules-->
             
             <xsl:call-template name="dataTypeIncorrectType">
                 <xsl:with-param name="dataTypeElement" select="."/>
             </xsl:call-template>
-            <xsl:call-template name="dataTypeUniqueName">
+<!--            <xsl:call-template name="dataTypeUniqueName">
```

Comment:
as seen in the changes in this pull request this was replaced by the new template elementUniqueName. It is commented out and not deleted just in case someone will want to revert the changes made in this pull request

Reply:
as seen in the changes in this pull request this was replaced by the new template elementUniqueName. It is commented out and not deleted just in case someone will want to revert the changes made in this pull request